### PR TITLE
Workaround solution for issue #349

### PR DESCRIPTION
**Pull request by diablero13**
Issue #349. Disabled switching of focus on the `<code/>` elements.

### CODE REVIEW

File: js/editable-css.js [hunk 1]
```diff
@@ -7,28 +7,29 @@
...
     var initialChoice = 0;
     var originalChoices = [];
-    var output = document.getElementById('output');
 
     /**
      * Enables and initializes the live code editor
      */
     function enableLiveEditor() {
+        var header = document.querySelector('header'),
+            output = document.getElementById('output');
```
Comment (schalkneethling) [inline js/editable-css.js]: Nit, code style: We do not use the single `var` pattern. Please declare each variable on its own i.e.

```
var header = document.querySelector('header');
var output = document.getElementById('output');
```

File: js/editable-css.js [hunk 2]
```diff
@@ -7,28 +7,29 @@
...
+
         header.classList.remove('hidden');
         exampleChoiceList.classList.add('live');
         output.classList.remove('hidden');
 
-        for (var i = 0, l = exampleChoices.length; i < l; i++) {
-            var exampleChoice = exampleChoices[i];
+        for (var i = 0, choiceContainer, choiceCode; i < exampleChoices.length; i++) {
```
Comment (schalkneethling) [inline js/editable-css.js]: Nit, code style: I would not suggest this code style, rather declare the variables inside the loop explicitly. Also, cache the length of the `Array` i.e.

```
for (var i = 0 l = exampleChoices.length, ; i < l; i++) {
    var choiceContainer = exampleChoices[i];
    var choiceCode = choiceContainer.querySelector('code');
```

**Comment by schalkneethling** [comment]
Hey @diablero13 

We really appreciate your contributions but, your current avatar(profile picture) goes against our code of conduct. Please change it to something more less offensive. It does not have to be a photo of yourself.

I hope you understand, and look forward to merging this and many more contributions from you. Thank you in advance.

**Comment by diablero13** [comment]
Hi @schalkneethling. Current pic is ok?

**Comment by schalkneethling** [comment]
> Hi @schalkneethling. Current pic is ok?

Great! Thanks so much for your understanding @diablero13

**Comment by schalkneethling** [comment]
@diablero13 Just checking in on this one. Are you still working on it? Thanks!

**Pull request closed without merging**